frontend/src/pages/Home/Home.tsx

```diff
@@ -14,26 +15,30 @@ interface PokemonCaracteristics {
 }
 interface State {
   pokemons: PokemonCaracteristics[];
+  loading: boolean;
 }
 
 class Home extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
       pokemons: [],
+      loading: true,
     };
   }
 
   componentDidMount() {
-    makeGetRequest('/pokemon').then(response => this.setState({ pokemons: response.body }));
+    makeGetRequest('/pokemon')
+      .then(response => this.setState({ pokemons: response.body }))
+      .then(_ => this.setState({ loading: false }));
```

Comment:
quand tu n'as pas d'argument à passer, utiliser () plutot que _
genre
.then(() => this.setState({ loading: false }))